src/client/entity-editor/name-section/name-section.js

```diff
@@ -139,7 +139,10 @@ class NameSection extends React.Component {
 								)}
 								error={!validateNameSectionName(nameValue)}
 								inputRef={this.updateNameFieldInputRef}
-								tooltipText={`Official name of the ${_.capitalize(entityType)} in its original language. Names in other languages should be added as 'aliases'.`}
+								tooltipText={
+								    `Official name of the ${_.capitalize(entityType)}
+									&nbsp; in its original language. Names in other languages should be added as 'aliases'.`
```

Comment:
It looks like you did not check this either. I think the tooltip does not parse html code and is considering it as a string:

<img width="1169" alt="Capture d’écran 2019-03-19 à 16 18 44" src="https://user-images.githubusercontent.com/6179856/54618189-e4f4a700-4a62-11e9-9ee9-70ddbe1c3069.png">